content/_index.pt.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Home
+mobile_menu_title: "Home"
+---
+{{< slogan get_started="COMECE AGORA" docs="Documentação" notes="Mudanças">}}
+Zig é uma linguagem de programação de propósito geral e um conjunto de ferramentas para manter o seu software **robusto**, **otimizado**, e **usável***.  
+{{< /slogan >}}
+
+{{< flexrow class="container" style="padding: 20px 0; justify-content: space-between;" >}}
+{{% div class="features" %}}
+
+# ⚡ Uma linguagem simples
+Concentre-se na depuração de sua aplicação em vez de depurar seus conhecimentos em uma linguagem de programação.
+
+- Sem fluxos de controle ocultas.
+- Sem alocações de memórias ocultas.
+- Sem preprocessador, nem macros. 
+
+# ⚡ Tempo de Compilação (Comptime)
+Uma nova abordagem de metaprogramação baseada na execução do código em tempo de compilação e avaliação preguiçosa.
+
+- Chame qualquer função em tempo de compilação.
+- Manipular tipos como valores sem sobrecarga de tempo de execução.
+- O Comptime emula a arquitetura alvo.
+
+# ⚡ Desempenho que se enquadra na segurança
+Escreva um código rápido e claro, capaz de lidar com todas as condições de erro.
+
+- A linguagem guia graciosamente sua lógica de tratamento de erros.
+- As verificações configuráveis de tempo de execução ajudam a encontrar um equilíbrio entre desempenho e garantias de segurança.
+- Aproveite os tipos vetoriais para expressar as instruções SIMD de forma portátil.
+
+{{% flexrow style="justify-content:center;" %}}
+{{% div %}}
+<h1>
+    <a href="learn/overview/" class="button" style="display: inline;">Visão geral em profundidade</a>
```

Comment:
Is it possible to shorten the text in this and/or the other button so that they can both fit on the same line when displayed on a desktop computer? 

If it's doable by tweaking the translation a little, then great; if not, then it's fine as is.